ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -60,3 +60,49 @@ def ui_deployment_conditions():
         return False
     else:
         return True
+
+
+def ui_add_capacity_conditions():
+    """
+    Conditions for add capacity via UI
+
+    return:
+        bool: True if support UI add capacity, False otherwise
+    """
+    platform = config.ENV_DATA["platform"]
+    ocp_version = get_ocp_version()
+    is_external = config.DEPLOYMENT["external_mode"]
+    is_disconnected = config.DEPLOYMENT.get("disconnected")
+    is_lso = config.DEPLOYMENT.get("local_storage")
+    http_proxy, https_proxy, no_proxy = get_cluster_proxies()
+    is_proxy = True if http_proxy else False
+
+    try:
+        locators[ocp_version]["add_capacity"]
+    except KeyError as e:
+        logger.info(
+            f"Add capacity via UI is not supported on ocp version {ocp_version}"
+        )
+        logger.error(e)
+        return False
+
+    if platform.lower() not in (constants.AWS_PLATFORM, constants.VSPHERE_PLATFORM):
+        logger.info(f"Add capacity via UI is not supported on platform {platform}")
+        return False
+    elif ocp_version not in ("4.7", "4.8"):
+        logger.info(
+            f"Add capacity via UI is not supported when the OCP version [{ocp_version}]"
+        )
+        return False
+    elif is_external or is_disconnected or is_proxy or is_lso:
+        if is_external:
+            logger.info("Add capacity via UI is not supported on external cluster")
+        if is_disconnected:
+            logger.info("Add capacity via UI is not supported on disconnected cluster")
```

Comment:
Shouldn't this message be "Add capacity via UI is not automated at the moment on disconnected cluster"?

Reply:
There is no difference between disconnected/external/proxy cluster and "regular" cluster (for UI automation), sometimes there is a SSL issue (at login UI step) on these type of clusters.

Reply:
Resolved. Requested change has been incorporated.